Sprintfinity3902/Game1.cs

```diff
@@ -107,15 +82,11 @@ public class Game1 : Game {
             SpriteBatch.Begin(SpriteSortMode.Deferred, null, SamplerState.PointClamp);
 
             Camera.Instance.Draw(SpriteBatch);
-            sprintTwo.Draw(SpriteBatch);
 
+            sprintTwo.Draw(SpriteBatch);
```

Comment:
sprintTwo is a really undescriptive name, I thought I grabbed most of these last night, but I would like to change them.